src/root/components/header.js

```diff
@@ -148,6 +148,7 @@ function Header (props) {
                 <li>
                   <NavLink to='/' title={strings.headerMenuHomeTooltip} activeClassName='navbar-highlighted' exact>
                     <Translate stringId='headerMenuHome' />
+                    { strings.headerMenuHome }
```

Comment:
This is causing Home to appear twice in the menu :)

Reply:
Oops! Was only meant for testing. Thanks for catching it :+1: